docs/ops/movement/Slice_8.md

```diff
@@ -0,0 +1,446 @@
+## Slice <a name="Slice"></a> {#openvino_docs_ops_movement_Slice_8}
+
+**Versioned name**: *Slice-8*
+
+**Category**: *Data movement*
+
+**Short description**: *Slice* operation extracts a slice of the input tensor.
+
+**Detailed Description**: *Slice* operation selects a region of values from the `data` tensor.
+Selected values start at indexes provided in the `start` input and end
+at indexes provides in `stop` input (exclusively).
+
+Input `step` allows subsampling of `data`, selecting evey *n*-th element,
+where `n` is equal to `step` element for corresponding axis.
+Negative `step` value indicates slicing backwards, so the sequence along the corresponding axis is reversed in the output tensor.
+
+Optional `axes` input allows specifying slice indexes only on selected axes.
+Other axes will not be affected and will be output in full.
+
+The rules follow python language slicing `data[start:stop:step]`.
+
+**Attributes**: *Slice* operation has no attributes.
+
+**Inputs**
+
+* **1**: `data` - tensor (to be sliced) of type *T* and shape rank greater or equal to 1. **Required.**
+
+* **2**: `start` - 1D tensor of type *T_INT*. Indices corresponding to axes in `data`.
+  Defines the starting coordinate of the slice in the `data` tensor.
+  A negative index value represents counting elements from the end of that dimension.
+  A value larger than the size of a dimension is silently clamped. **Required.**
+
+* **3**: `stop` - 1D, type *T_INT*, similar to `start`.
+  Defines the coordinate of the opposite vertex of the slice, or where the slice ends.
+  Stop indexes are exclusive, which means values lying on the ending edge are
+  not included in the output slice.
+  To slice to the end of a dimension of unknown size `INT_MAX`
+  may be used (or `INT_MIN` if slicing backwards). **Required.**
+
+* **4**: `step` - 1D tensor of type *T_INT* and the same shape as `start` and `stop`.
+  Integer value that specifies the increment between each index used in slicing.
+  Value cannot be `0`, negative value indicates slicing backwards. **Required.**
+
+* **5**: `axes` - 1D tensor of type *T_INT*.
+  Optional 1D tensor indicating to which dimensions apply the values in `start` and `stop`.
+  Negative value means counting dimensions from the end. The range is `[-r, r - 1]`, where `r` is the rank of the `data` input tensor.
+  Values are required to be unique. If a particular axis is unspecified, it is considered as a whole dimension.
+  Default value: `[0, 1, 2, ..., start.shape[0] - 1]`. **Optional.**
+
```

Comment:
also I think that types of `start` and `axes` can be different. So please use `T_AXIS` for axes input

Reply:
Done